core/src/main/java/com/alibaba/nacos/core/file/WatchFileCenter.java

```diff
@@ -176,7 +175,8 @@ void addSubscribe(final FileWatcher watcher) {
 		}
 
 		void shutdown() {
-			watch = false;
+		    watch = false;
+            ThreadUtils.shutdownThreadPool(this.callBackExecutor);
```

Comment:
If register to managed, whether should shutdown in here?

Reply:
yes。if register to managed